### ISSUE TITLE
Subtasks should be listed in Backlog

### ISSUE THREAD

**Issue by Akasch** [256MbTeam/Redmine-Scrumbler#68]
First: Thanks great Plugin 

Second: Description of Situation:

We use this Plugin for showing Storys (Root Ticket) and Tasks (Sub Tickets of a Specific Story). Assume we create a Ticket with the Tracker Story and add two Subtickets with the Tracker Task. Then we configured the Aviable Trackers in Backlogs and Sprint as Task. In this SetUp if we choose the Backlog overview we don't see any Tasks. 

In our Process we create Storys and collect them as Backlog Items. Before we start a Sprint we have created Subtasks as Tasks of that Storys. 

Our Vison:

The best things would be, if it is possible to move the Tasks of a Story separately from each other but have a color that applies to the Story and all of the Subtasks belonging to them.
(Screenshot of the Vision http://images.mercarion-online.de/scrumbler_vision.png)

This is our long term request. As first (hopefully) Quick Solution it would be nice if just subtasks would be shown in Backlog.
(as described above) 

A reply if you like/add/deny this feature would be nice
Greetings
Team Hamburg Bit-Bots (bit-bots.de)

**Comment by sheepsy90** [comment]
That seems cool - would be easier to manage Storys and resulting tasks :)

+1

**Comment by hawkhead99** [comment]
I agree, this would solve our current problem managing subtasks and I like the idea of it being in a hierarchy in the long term

+1

**Comment by StewyH** [comment]
I  have already added a request see.  https://github.com/256MbTeam/Redmine-Scrumbler/issues/54
I think a lot of users  would appreciate this feature
I do like your suggested enhancement(s) 

+1

**Comment by pprkut** [comment]
+1

**Comment by mlocher** [comment]
+1

**Comment by RohitTalwalkar-zz** [comment]
+1 to this as well, would be a great feature.

**Comment by monofonik** [comment]
Definitely need this... is there a workaround for this at the moment? We need to handle sub-tasks separately on the scrum board.

**Comment by hawkhead99** [comment]
What is this?

Scott Brazelton

-------- Original message --------
Subject: New Ticket Alert
From: tickets
To: Development
CC:

Help Desk,

## New ticket #494640 created.

Name: Rohit Talwalkar
Email: rtalwalkar@centradhealthcare.com
Dept: Help Desk

-----Original Message-----
From: service@gts.globalec.com [mailto:service@gts.globalec.com]
Sent: Friday, September 21, 2012 5:24 PM
To: Rohit Talwalkar
Subject: 030 Report A611731200010001

---030---                IBM(r) STERLING INFORMATION BROKER(r)                R0002
                            TRANSMISSION ERROR REPORT

CPU:  21 SEP 12  18:24:09                                               PAGE:  1
GMT:  21 SEP 12  22:24:09

MAILBOX NUMBER:  SYPKMASC       COMPANY NAME:  CENTRAD HEALTHC
BATCH NUMBER:    O0102173

------------------CONTROL NUMBERS----------------- --------DOCUMENT------------
    INTERCHANGE      GROUP            DOCUMENT       SEGS    CHARS     TYPE

---

   000051436        51436           514360001          503      7589 PT  867
   Business Doc #   51436
_WARNING_SE    REQUIRED SEGMENT MISSING

S/R  6303579ASC                           008256224ANCN
   000051437        51437                              126      2105 PT  867
   Business Doc #
_WARNING_ST    REQUIRED SEGMENT MISSING
_**0404_*TRANSACTION SET SEGMENT COUNT (SE01) DOES NOT MATCH ACTUAL COUNT

TRANSMISSION TOTALS
    DOCUMENTS:       2   DOCUMENT CHARS:       9694     DOCUMENT SEGS:      629
                         ENVELOPE CHARS:        382     ENVELOPE SEGS:        8
                         TOTAL CHARS:         10076     TOTAL SEGS:         637

---

To view/respond to the ticket, please login to the support ticket system.

http://cats.centrad.net/scp/
- Your friendly CATS Request service
